index.coffee

```diff
@@ -43,20 +40,20 @@ router.get '/', (ctx)->
   """
 
 router.get  '/week', week = (ctx)->
-  events = await this_week()
+  events = await Event.this_week()
   ctx.assert events.length, 404, 'No events this week'
   ctx.body = slack_payload events, 'Events this week'
 
 router.post '/week', (ctx)->
   await post_to_slack await week ctx
 
 router.get  '/today', today = (ctx)->
-  events = await today()
+  events = await Event.today()
   ctx.assert events.length, 404, 'No events today'
   ctx.body = slack_payload events, 'Events today'
 
 router.post '/today', (ctx)->
-  await post_to_slack await today ctx
+  await post_to_slack await Event.today ctx
```

Comment:
this actually needs to call the `today` method defined on L50, since it passes the body to `post_to_slack`

Reply:
Oh oops, you are right! Fixed :+1: